### PR TITLE
XCreds 3.1 update

### PR DESCRIPTION
**Pull request by twocanoes**
New keys for XCreds 3.1 update

### CODE REVIEW

File: Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist [hunk 1]
```diff
@@ -607,6 +845,6 @@ A profile can consist of payloads with different version numbers. For example, c
 	<key>pfm_unique</key>
 	<false/>
 	<key>pfm_version</key>
-	<integer>3</integer>
+	<integer>5084</integer>
```
Comment (kevinmcox) [inline Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist:848]: `pfm_version` refers to the version of this file. So this update is revision `4`.

Comment (twocanoes) [reply Manifests/ManagedPreferencesApplications/com.twocanoes.xcreds.plist]: Fixed pfm_version

**Comment by twocanoes** [comment]
Anything else we need to do?

**Comment by twocanoes** [comment]
bump, rumble rumble, bump :)

**Review by kevinmcox — APPROVED** [review]
LGTM, but @apizz usually spots things I don't.

**Comment by twocanoes** [comment]
Is this merged in? We are not seeing it yet. Please merge :)

**Review by apizz — APPROVED** [review]
LGTM.  Sorry for the delay

**Pull request merged**